container/kubernetes/onboarding/config.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This file is going to be removed?

Reply:
We need to provide a template of this file ?

Reply:
@wagmarcel : Do i need to remove config.json file ?  Don't we need to provide them the template as part of this repo and it can be either replaced from IndustryFusion repo config.json or created there own based on this template ?

Reply:
I am removing this file from repo for now and pushing the code for review.